plugins/inputs/cisco_telemetry_mdt/README.md

```diff
@@ -21,6 +21,9 @@ The TCP dialout transport is supported on IOS XR (32-bit and 64-bit) 6.1.x and l
  ## Address and port to host telemetry listener
  service_address = ":57000"
 
+ ## Grpc Maximum Message Size, default is 4MB, increase the size.
+ # max_msg_size = 20000000
```

Comment:
According to https://github.com/influxdata/telegraf/wiki/SampleConfig the commented-out option should have the default value.